rules/windows/process_creation/proc_creation_win_mode_codepage_russian.yml

```
@@ -0,0 +1,40 @@
+title: CodePage Modification Via MODE.COM To Russian Language
+id: 12fbff88-16b5-4b42-9754-cd001a789fb3
+related:
+    - id: d48c5ffa-3b02-4c0f-9a9e-3c275650dd0e
+      type: derived
+status: experimental
+description: |
+    Detects a CodePage modification using the "mode.com" utility to Russian language.
+    This behavior has been used by threat actors behind Dharma ransomware.
+references:
+    - https://learn.microsoft.com/en-us/windows/win32/intl/code-page-identifiers
+    - https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/mode
+    - https://strontic.github.io/xcyclopedia/library/mode.com-59D1ED51ACB8C3D50F1306FD75F20E99.html
+    - https://www.virustotal.com/gui/file/5e75ef02517afd6e8ba6462b19217dc4a5a574abb33d10eb0f2bab49d8d48c22/behavior
+author: Joseliyo Sanchez, @Joseliyo_Jstnk
+date: 2024/01/17
+tags:
+    - attack.defense_evasion
+    - attack.t1036
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    # VT Query: behavior:"mode con cp select=1251"
+    # VT Query: behavior:"mode con cp select=866"
+    selection_img:
+        - Image|endswith: '\mode.com'
+        - OriginalFileName: 'MODE.COM'
+    selection_cli:
+        CommandLine|contains|all:
+            - ' con '
+            - ' cp '
+            - ' select='
+        CommandLine|endswith:
+            - '1251' # ANSI Cyrillic; Cyrillic (Windows) - Observed ITW by Dharma ransomware
+            - '866' # OEM Russian; Cyrillic (DOS) - Observed ITW by other malware
```

```suggestion
            - '=1251' # ANSI Cyrillic; Cyrillic (Windows) - Observed ITW by Dharma ransomware
            - '=866' # OEM Russian; Cyrillic (DOS) - Observed ITW by other malware
```
should be fine and more strict like this